fdbclient/admin_client.go

```diff
@@ -332,17 +332,19 @@ func (client *cliAdminClient) GetExclusions() ([]fdbv1beta2.ProcessAddress, erro
 
 // getRemainingAndExcludedFromStatus checks which processes of the input address list are excluded in the cluster and which are not.
 // The first list return all addresses that are excluded in the cluster and the exclude command can be used to verify if it's safe to remove this address.
-// The second list contains all addresses that are part of the input list and are not marked as excluded in the cluster, those addresses are not safe to remove.
-func getRemainingAndExcludedFromStatus(status *fdbv1beta2.FoundationDBStatus, addresses []fdbv1beta2.ProcessAddress) ([]fdbv1beta2.ProcessAddress, []fdbv1beta2.ProcessAddress) {
-	notExcluded := map[string]fdbv1beta2.None{}
+// The second list returns all addresses that are excluded and don't have any roles assigned, this is a sign that the process is "fully" excluded and safe to remove.
+// The third list contains all addresses that are part of the input list and are not marked as excluded in the cluster, those addresses are not safe to remove.
+func getRemainingAndExcludedFromStatus(status *fdbv1beta2.FoundationDBStatus, addresses []fdbv1beta2.ProcessAddress) ([]fdbv1beta2.ProcessAddress, []fdbv1beta2.ProcessAddress, []fdbv1beta2.ProcessAddress) {
```

Comment:
Since we're now adding a third return value with the same type, it may be worth defining a new type to capture the returned address lists, to make it more explicit in the code what each one means.